src/nb_enum.cpp

```diff
@@ -1,5 +1,5 @@
 #include "nb_internals.h"
-
+#include <string>
```

Comment:
Can we avoid depending on `std::string`? I've tried hard not to have anything depend on it in the core library.